css/style.css

```diff
@@ -1,289 +0,0 @@
-  margin-bottom: 8rem;
-}
-
-.cards-mobile {
-  background-image: url("/images/bg-main-mobile.png");
-  height: 100%;
-  width: 100%;
-}
-
-.card {
-  background-repeat: round;
-  position: relative;
-  width: 70%;
-  margin: 0 auto;
-  height: 55%;
-  padding: 1.5rem;
-}
-
-.card-front {
-  background-image: url("/images/bg-card-front.png");
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  right: 1.4rem;
-  bottom: 2.3rem;
-  border-radius: 0.5rem;
-  z-index: 2;
-}
-
-.card-logo {
-  width: 3rem;
-}
-
-.card-number {
-  color: var(--White);
-  margin-top: 1.5rem;
-  letter-spacing: 0.2rem;
-  font-size: 1.1rem;
-}
-
-.card-info {
-  display: flex;
-  justify-content: space-between;
-  color: var(--White);
-  font-size: 0.75rem;
-}
-
-.card-back {
-  background-image: url("/images/bg-card-back.png");
-  top: 2.6rem;
-  left: 1.5rem;
-  z-index: 1;
-}
-
-.security-number {
-  display: flex;
-  justify-content: right;
-  position: relative;
-  right: 0.5rem;
-  top: 45%;
-  font-size: 0.75rem;
-  color: var(--White);
-}
-
-.info-section {
-  height: calc(100% - 26% - 8rem - 1rem);
-}
-
-form,
-.thank-you-state {
-  width: 90%;
-  height: 100%;
-  margin: auto;
-}
-
-.provide-info {
-  display: flex;
-  flex-direction: column;
-  gap: 3rem;
-  margin-bottom: 4rem;
-}
-
-.provide-name,
-.provide-card-number,
-.provide-sec-info {
-  display: flex;
-  flex-direction: column;
-  gap: 0.5rem;
-}
-
-.provide-info label {
-  font-size: 0.75rem;
-  letter-spacing: 0.1rem;
-  color: var(--Very-dark-violet);
-  font-weight: 600;
-  margin-bottom: 1rem;
-}
-
-.provide-info input {
-  padding: 0.75rem;
-  border-radius: 0.5rem;
-  border: 1px var(--Light-Grayish-Violet) solid;
-}
-
-.provide-info input::placeholder {
-  color: var(--Dark-Grayish-Violet);
-}
-
-.provide-info input::-webkit-inner-spin-button {
-  appearance: none;
-}
-
-.provide-info input:focus {
-  outline: 1px var(--Purple) solid;
-}
-
-.error {
-  outline: 1px var(--Error-Red) solid;
-}
-
-.error-msg {
-  font-size: 0.75rem;
-  color: var(--Error-Red);
-}
-
-.provide-sec-info {
-  flex-direction: row;
-  gap: 0.4rem;
-}
-
-.exp-date,
-.provide-cvc {
-  display: flex;
-  flex-direction: column;
-  gap: 1rem;
-}
-
-.exp-date-details {
-  display: flex;
-  gap: 0.5rem;
-}
-
-.exp-date-details > input {
-  width: 35%;
-}
-
-.btn {
-  background-color: var(--Very-dark-violet);
-  color: var(--White);
-  width: 100%;
-  padding: 1.25rem;
-  border: none;
-  border-radius: 0.5rem;
-  font-size: 1rem;
-}
-
-.thank-you-state {
-  display: none;
-  flex-direction: column;
-  align-items: center;
-  gap: 5rem;
-}
-
-.thank-you-info {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  gap: 2rem;
-  text-align: center;
-}
-
-.thank-you-heading {
-  font-size: 2rem;
-}
-
-.thank-you-msg {
-  color: var(--Dark-Grayish-Violet);
-}
-
-.attribution {
-  position: absolute;
-  height: 1rem;
-  font-size: 0.75rem;
-  text-align: center;
-  width: 100%;
```

Comment:
**question:** If it's div there is no need to add width: 100%; because it's default value. But I'm not sure how does it work with position: absolute;

Could you try this?

Reply:
Yes it works, but only on desktop and while I have it on mobile view it comes to the corner so I need to make it 100% so the  text-align could work.